### PR TITLE
Getting to 100% line coverage in tests.

### PR DESCRIPTION
**Pull request by dhermes**

### CODE REVIEW

File: tests/test_client.py [hunk 1]
```diff
@@ -357,17 +347,16 @@ def test_save_to_well_known_file_service_account(self):
         self.assertEqual('ABCDEF', d['private_key_id'])
         os.remove(temp_credential_file)
 
-    def test_save_well_known_file_with_non_existent_config_dir(self):
+    @mock.patch('os.path.isdir', return_value=False)
+    def test_save_well_known_file_with_non_existent_config_dir(self,
```
Comment (nathanielmanistaatgoogle) [inline tests/test_client.py:351]: I don't understand the change on this line. Mustn't `self` be the exactly one parameter of a test method?

File: tests/test_file.py [hunk 2]
```diff
@@ -177,7 +177,7 @@ def test_token_refresh_good_store(self):
         new_cred.access_token = 'bar'
         s.put(new_cred)
 
-        credentials._refresh(lambda x: x)
+        credentials._refresh(None)
```
Comment (nathanielmanistaatgoogle) [inline tests/test_file.py:180]: What's the reason for this change?

File: tests/test_client.py (hunk 1 above)
Comment (dhermes) [inline tests/test_client.py:351]: `@mock.patch` (which is now in the standard library in Python 3) adds sugar for more arguments. Also, I've used it in PRs that you've reviewed before.

File: tests/test_file.py (hunk 2 above)
Comment (dhermes) [inline tests/test_file.py:180]: The `good_store` part of `test_token_refresh_good_store` means the refresh method is not needed, so `coverage` was (correctly) flagging `lambda x: x` as something which was never executed.

File: tests/test_client.py (hunk 1 above)
Comment (nathanielmanistaatgoogle) [inline tests/test_client.py:351]: Alright, I suppose I'll have to go off and learn it then.

Comment (dhermes) [inline tests/test_client.py:351]: It's very useful. It took me awhile but I'm happy I did.
